File: csharp/DVSClient/DVSClient/address/Accuracy.cs
using DVSClient.Common;

namespace DVSClient.Address
{
    public enum Accuracy
    {
        [EnumStringValue("A+")]
        APlus = 1,
        [EnumStringValue("A")]
        A = 2,
        [EnumStringValue("A-")]
        AMinus = 3,
        [EnumStringValue("B")]
        B = 4,
        [EnumStringValue("B-")]
        BMinus = 5
    }
}

File: java/src/main/java/com/experian/dvs/client/address/Accuracy.java
package com.experian.dvs.client.address;

public enum Accuracy {
    A_PLUS("A+"),
    A("A"),
    A_MINUS("A-"),
    B("B"),
    B_MINUS("B-")
    ;

    private final String value;

    Accuracy(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}
